### ISSUE TITLE
Events without icon show broken image in chrome

### ISSUE THREAD

**Issue by friedger** [gdg-x/boomerang#29]
If an event does not have an event.IconUrl a broken image is shown in chrome, or alt text "GDG Boomerang" is shown.

Should use the default icon!

**Comment by Splaktar** [comment]
I've created a pull request that should fix this, but I can't reproduce so I will need you to verify it. Thanks.
